src/main/java/li/cil/tis3d/common/capabilities/Capabilities.java

```diff
@@ -2,22 +2,13 @@
 
 import li.cil.tis3d.api.infrared.InfraredReceiver;
 import net.minecraftforge.common.capabilities.Capability;
-import net.minecraftforge.common.capabilities.CapabilityInject;
-import net.minecraftforge.common.capabilities.CapabilityManager;
+import net.minecraftforge.common.capabilities.RegisterCapabilitiesEvent;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
 
 public final class Capabilities {
-    @CapabilityInject(InfraredReceiver.class)
     public static Capability<InfraredReceiver> INFRARED_RECEIVER = null;
```

Comment:
From my understanding, this should now be initialized with `CapabilityManager.get(new CapabilityToken<>() { })`. If you rebase onto/merge the updated 1.17 branch, that's already in there (will mean minor conflicts, sorry for that, but the deprecation warnings bothered me :x).

Reply:
hey, the deprecation warnings bothered me too, but i was trying to limit how much i was caring about at the time. it's worth the extra time to manage conflicts. :)